Paintroid/src/main/java/org/catrobat/paintroid/common/MainActivityConstants.java

```diff
@@ -46,6 +46,7 @@
 	public static final int PERMISSION_EXTERNAL_STORAGE_SAVE_CONFIRMED_NEW_EMPTY = 4;
 	public static final int PERMISSION_EXTERNAL_STORAGE_SAVE_CONFIRMED_FINISH = 5;
 	public static final int PERMISSION_REQUEST_CODE_LOAD_PICTURE = 6;
+	public static final int PERMISSION_REQUEST_CODE_IMPORTPNG = 7;
```

Comment:
```suggestion
	public static final int PERMISSION_REQUEST_CODE_IMPORT_PICTURE = 7;
```
All image formats are supported when importing. Please change everywhere.

Reply:
done